calculator.py

```diff
@@ -108,10 +115,13 @@ def divide(self,a: float,b: float) -> float:
 
 
 def start_calculator():
-    args = create_parser()
-    salusion=take_expresion(args.f)
-    cal= Calculator()
-    print(cal.calculate(salusion))
+    with measure_time() as start_time:
```

Comment:
This isn't a timer, it just returns current time. Try to make it return the elapsed time when existing the context manager.
Something along the lines of:
```
with measure_time() as timer:
    do
    code
print(timer.get_elapsed_time())